UnityProjects/MRTKDevTemplate/Assets/Data Binding Example/Scripts/DataSourceDynamicList.cs

```diff
@@ -85,9 +85,7 @@ protected void Randomize(string[] items)
             for (int i = 0; i < items.Length - 1; i++)
             {
                 int j = rand.Next(i, items.Length);
-                string temp = items[i];
-                items[i] = items[j];
-                items[j] = temp;
+                (items[j], items[i]) = (items[i], items[j]);
```

Comment:
:O c# is so cool

Reply:
Yeah, I didn't know this was a thing until Visual Studio recommended it haha! I want to use it everywhere now 😂